casanovo/casanovo.py

```diff
@@ -354,9 +355,10 @@ def setup_model(
     seed_everything(seed=config["random_seed"], workers=True)
 
     # Download model weights if these were not specified (except when training).
+    cache_dir = appdirs.user_cache_dir("casanovo", False, opinion=False)
```

Comment:
**suggestion:** Convert this to a `Path` already here and then have all functions that do something with the cache dir have a consistent input argument (instead of `str` vs `Path` currently).